examples/advanced/llm_hf/README.md

```diff
@@ -99,55 +83,37 @@ The major code modifications are for receiving and returning the global model (r
 ![diff](./figs/diff_fl_1.png)
 ![diff](./figs/diff_fl_2.png)
 
-## Job for NVFlare FL Training
-With the local training script ready, we can go ahead to generate the NVFlare job configs by reusing the job templates from [sag_pt](../../../job_templates/sag_pt/).
-
-Let's set the job template path with the following command.
-```bash
-nvflare config -jt ../../../job_templates/
-```
-Then we can check the available templates with the following command.
-```bash
-nvflare job list_templates
-```
-We can see the "sag_pt" template is available, with which we further generate job configs for SFT and PEFT as:
-```
-nvflare job create -force -j "./jobs/hf_peft" -w "sag_pt" -sd "code" \
-  -f meta.conf min_clients=1 \
-  -f config_fed_client.conf app_script="hf_sft_peft_fl.py" app_config="--model_path ${PWD}/model/Llama-2-7b-hf --data_path_train ${PWD}/dataset/dolly/training.jsonl --data_path_valid ${PWD}/dataset/dolly/validation.jsonl --output_path llama2-7b-dolly-peft --mode 1" \
-  -f config_fed_server.conf model_class_path="hf_peft_model.CausalLMPEFTModel" components[0].args.model.args.model_path="${PWD}/model/Llama-2-7b-hf" min_clients=1 num_rounds=3 key_metric="eval_loss" negate_key_metric=True
+### Federated Training Results
+We run the federated training on a single client using NVFlare Simulator via [JobAPI](../job_api/README.md).
```

Comment:
better link to the docs https://nvflare.readthedocs.io/en/main/programming_guide/fed_job_api.html

Reply:
thanks, updated